extra/2-piping.js

```diff
@@ -16,26 +16,41 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a,b) {
+  return +(a + b).toFixed(1);
```

Comment:
Great job! 
I think the task was to give the total No 2 decimals .toFixed(2)

Reply:
Hi hi, thank you. But my interpretation of this test: "test("add function - case 2 works", () => {
  expect(add(2.4, 5)).toEqual(7.4);
})"  is that I should ensure that there is one decimal place and not 2 for example if I use .toFixed(2) as you suggested, my answer may end up like 7.40 instead of 7.4 as an example left in the test. 

I could be wrong but that is how I looked at it. 

Also if I did not put .toFixed(1) I would get the recommended 7.4  and 10.1 but again I am not too sure what they expected hence I added .toFixed(1)